src/Model/LandingPageRepository.php

```diff
@@ -171,8 +176,10 @@ public function deleteById(int $pageId): bool
      */
     public function findAllActive(): array
     {
+        $storeId = $this->storeManager->getStore()->getId();
+
         $searchCriteria = $this->searchCriteriaBuilder
-            ->addFilter(LandingPageInterface::ACTIVE, 1)
+            ->addFilter(LandingPageInterface::STORE_IDS, ['in' => $storeId])
```

Comment:
@ah-net How was this PR merged and released? 
This is not working at all, it will raise a syntax error in the SQL query.
The signature of `addFilter` method is as follows:

```php
public function addFilter($field, $value, $conditionType = 'eq')
```

Also the active flag still must be checked as well, as that allows to temporarily deactive a landing page.

Also you can set stores to 0 for all stores, so that one also needs to be checked.

I will make another PR with the fixes.

Reply:
@bramstroker You're right, seems i missed that. I think I missed this because the changes where suggested by somebody else and I just checked if the result was wat i expected. I'll double check things in the future.

Weird thing is, i tested this with generating an sitemap with attribuutlanding pages. And got the expected result instead of alle attribute landingpages for all shops. And didn't get an sql error (still don't). It's possible that you only get the error with specific combination of settings.